apps/web/app/app.dub.co/(dashboard)/[slug]/(ee)/settings/security/saml.tsx

```
@@ -197,40 +199,40 @@ export function SAML() {
-              loading={isLoading}
-              disabled={plan !== "enterprise"}
-              fn={handleSSOEnforcementChange}
-            />
           </div>
         </div>
 
-        <div className="rounded-b-lg border-t border-neutral-200 bg-neutral-50 px-3 py-5 sm:px-10">
+        <div className="rounded-b-xl border-t border-neutral-200 bg-neutral-50 px-5 pb-4 pt-3">
           <a
             href="https://dub.co/help/category/saml-sso"
             target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)